tests/test_client/tests.py

```diff
@@ -1010,6 +1010,11 @@ async def test_follow_parameter_not_implemented(self):
                 with self.assertRaisesMessage(NotImplementedError, msg):
                     await method('/redirect_view/', follow=True)
 
+    async def test_get_data(self):
+        response = await self.async_client.get('/get_view/', {'var': 'val'})
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.content, b'This is a test. val is the value.')
```

Comment:
You can use `self.assertContains()`:
```suggestion
        self.assertContains(response, 'This is a test. val is the value.')
```

Reply:
Suggestion applied. Thanks a lot